### ISSUE TITLE
Gaps in stream occur during song change

### ISSUE THREAD

**Issue by chance-nelson** [chance-nelson/SnowStorm#21]
Depending on the size of the next song, there is occasionally a gap in the MP3 stream data, due to the API waiting for a song to be loaded. This can likely be alleviated by pre-loading the next song in the queue while another song is playing.

**Comment by gunjanraval** [comment]
Getting a 404 error on starting the server, any idea?

**Comment by sakethramanujam** [comment]
@gunjanraval do you have an environment variable  $MUSIC_LIBRARY set to use a specific set of music files?

**Comment by gunjanraval** [comment]
I actually got it working with some troubleshooting!

**Comment by chance-nelson** [comment]
Hi @suryakantpandey, yes, `$MUSIC_LIBRARY` points to a directory that is populated with your choice of MP3 files

**Comment by chance-nelson** [comment]
@gunjanraval Out of curiosity, can you describe what your error was and your solution? Sorry, I was away for a while

**Comment by gunjanraval** [comment]
yeah, part of the problem was I didn't have the variable MUSIC_LIBRARY set to the song files, which 
I was able to solve and the other was some package version dependency error, which also I solved using correct package versions from the requirements file.
